gateway/Gopkg.toml

```diff
@@ -12,7 +12,8 @@
 
 [[constraint]]
   name = "github.com/openfaas/nats-queue-worker"
-  version = "0.8.2"
+  source = "https://github.com/bmcstdio/nats-queue-worker"
```

Comment:
This will have to be changed once https://github.com/openfaas/nats-queue-worker/pull/79 gets merged.